stories/Menu.js

```diff
@@ -40,48 +40,60 @@ stories
         '', 'top-left', 'top', 'top-right', 'bottom-left', 'bottom', 'bottom-right'
       ], '') || undefined}
       size={select('Size', ['', 'small', 'medium', 'large'], '') || undefined}
+      onMenuClick={action()}
+      isOpen={boolean('isOpen', true)}
     >
-      <MenuDropdownList>
-        <MenuDropdownListItem
-          divider={boolean('Divider item 1', false)}
-        >Item 1</MenuDropdownListItem>
-        <MenuDropdownListItem
-          divider={boolean('Divider item 2', false)}
-        >Item 2</MenuDropdownListItem>
-        <MenuDropdownListItem
-          divider={boolean('Divider item 3', false)}
-        >Item 3</MenuDropdownListItem>
-        <MenuDropdownListItem
-          divider={boolean('Divider item 4', true)}
-        >Important last item</MenuDropdownListItem>
-      </MenuDropdownList>
-    </Menu>
+      <MenuSubHeader>
+        SubHeading 1
+      </MenuSubHeader>
+      <MenuItem
+        divider={boolean('Divider item 1', false)}
+        onClick={action('clicked menu item 1')}
+      >Item 1</MenuItem>
+      <MenuItem
+        divider={boolean('Divider item 2', false)}
+        onClick={action('clicked menu item 2')}
+      >Item 2</MenuItem>
+      <MenuSubHeader>
+        SubHeading 2
+      </MenuSubHeader>
+      <MenuItem
+        divider={boolean('Divider item 3', false)}
+        onClick={action('clicked menu item 3')}
+      >Item 3</MenuItem>
+      <MenuItem
+        divider={boolean('Divider item 4', true)}
+        onClick={action('clicked menu item 4')}
+      >Important last item</MenuItem>
+    </ControlledMenu>
   ))
-  .add('Custom Button', () => (
+  .add('Uncontrolled', () => (
     <Menu
-      customButton={customButton}
+      button={object('Button', button)}
       disabled={boolean('Disabled', false)}
-      isOpen={boolean('isOpen', false)}
       last={boolean('Last', false)}
-      nubbin={boolean('Nubbin', false)}
+      nubbin={boolean('Nubbin', true)}
       position={select('Position', [
         '', 'top-left', 'top', 'top-right', 'bottom-left', 'bottom', 'bottom-right'
       ], '') || undefined}
       size={select('Size', ['', 'small', 'medium', 'large'], '') || undefined}
+      isOpen={boolean('isOpen', false) || undefined}
```

Comment:
Should be renamed to `defaultOpen` for the uncontrolled menu since it should not do anything after the first click